### PR TITLE
Added Portuguese link and "contents"

### PR DESCRIPTION
**Pull request by alexmoreno**

### CODE REVIEW

**Comment by hemanth** [comment]
Yup

**Comment by alexmoreno** [comment]
I'm sorry, above which comment?

File: readme.md [hunk 1]
```diff
@@ -4,11 +4,20 @@ Functional programming (FP) provides a lot of advantages and its popularity has
...
 Where applicable, this document uses terms defined in the [Fantasy Land spec](https://github.com/fantasyland/fantasy-land)
 
+### Languages
+* [Portuguese](https://github.com/alexmoreno/jargoes-programacao-funcional)
+
+
+
+
+
 <!-- RM(noparent,notop) -->
```
Comment (felipemfp) [inline readme.md]: @alexmoreno maybe this one

Comment (alexmoreno) [inline readme.md]: @felipempf yes, but put it above what?

Comment (felipemfp) [inline readme.md]: @alexmoreno, `### Contents`. Because they are using [roadmarks](https://github.com/sielay/roadmarks)

File: readme.md [hunk 2]
```diff
@@ -4,12 +4,19 @@ Functional programming (FP) provides a lot of advantages and its popularity has
 
 JavaScript is popular language that is well suited to FP; especially with revision ES2015.  Therefore we have chosen to use JavaScript (ES2015) for our examples, and in some cases definitions.  This should make this glossary as accessible and as possible and introduce features from the new revision.
 
+
 *This is a [WIP](https://github.com/hemanth/functional-programming-jargon/issues/20); please feel free to send a PR ;)*
 
 Where applicable, this document uses terms defined in the [Fantasy Land spec](https://github.com/fantasyland/fantasy-land)
 
+### Languages
```
Comment (hemanth) [inline readme.md]:
```
__Tranlsations:__ 

* [Portuguese](https://github.com/alexmoreno/jargoes-programacao-funcional)
```

Looks good, thanks!

Comment (alexmoreno) [inline readme.md]: :P

Comment (hemanth) [inline readme.md]: Waiting for the changes ;)

**Comment by hemanth** [comment]
🐾

**Pull request merged**